pkg/kafka/policy.go

```diff
@@ -22,6 +22,53 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// isTopicAPIKey returns true if kind is apiKey message type which contains a
+// topic in its request.
+func isTopicAPIKey(kind int16) bool {
+	switch kind {
+	case api.ProduceKey:
+	case api.FetchKey:
+	case api.OffsetsKey:
+	case api.MetadataKey:
+	case api.LeaderAndIsr:
+	case api.StopReplica:
+	case api.UpdateMetadata:
+	case api.OffsetCommitKey:
+	case api.OffsetFetchKey:
+	case api.FindCoordinatorKey:
+	case api.JoinGroupKey:
+	case api.CreateTopicsKey:
+	case api.DeleteTopicsKey:
+	case api.DeleteRecordsKey:
+	case api.OffsetForLeaderEpochKey:
+	case api.AddPartitionsToTxnKey:
+	case api.WriteTxnMarkersKey:
+	case api.TxnOffsetCommitKey:
+	case api.AlterReplicaLogDirsKey:
+	case api.DescribeLogDirsKey:
+	case api.CreatePartitionsKey:
+		return true
+	}
+	return false
+}
+
+func matchNonTopicRequests(req *RequestMessage, rule api.PortRuleKafka) bool {
+	// These are non-topic requests. matchNonTopicRequests() is called when
```

Comment:
Please delete the first sentence in the comment here. It is misleading to me.

Reply:
Removed `These are non-topic requests.`